internal/testutil/testing.go

```diff
@@ -2,9 +2,16 @@ package testutil
 
 import (
 	"bytes"
+	"encoding/json"
 	"fmt"
 	"io"
+	"net/http/httptest"
 	"os"
+	"testing"
+
+	"gotest.tools/assert"
```

Comment:
Please use `github.com/stretchr/testify` instead for fancy assertations. It is already installed and used in other tests.

Reply:
As it turns out, `github.com/stretchr/testify` already has a method to compare JSON (oops). Implemented in 1fa7556